libs/ledgerjs/packages/hw-transport-http/src/WebSocketTransport.ts

```diff
@@ -109,7 +109,7 @@ export default class WebSocketTransport extends Transport {
         reject(e);
       }
     });
-    return new WebSocketTransport(exchangeMethods);
+    return new this(exchangeMethods);
```

Comment:
Can you add a comment explaining
> Since the open() is a static method that return the instance of the class and given WebSocketTransport can be extended (in our case by VaultTransport.ts) we want to return an instance of the good class and not always the parent one

pls ? :)